src/agent/api/testmanagementapi.ts

```diff
@@ -23,18 +23,18 @@ export class TestManagementApi implements ifm.ITestManagementApi {
     }
     
     public createTestRun(testRun: ifm.TestRun, onResult: (err: any, statusCode: number, publishedTestRun: ifm.TestRun) => void): void {
-        this.restClient.create('/_apis/test/runs', "3.0-preview.1", testRun, null, onResult);
```

Comment:
That would be great, ill undo this in the next iteration which will help in a clean merge